resources/services/codepipeline/gen.hcl

```diff
@@ -1,11 +1,16 @@
+//check-for-changes
 service          = "aws"
 output_directory = "."
 add_generate     = true
 
+description_modifier "remove_read_only" {
+  words = ["  This member is required."]
+}
+
 resource "aws" "codepipeline" "pipelines" {
   path = "github.com/aws/aws-sdk-go-v2/service/codepipeline.GetPipelineOutput"
   ignoreError "IgnoreAccessDenied" {
-    path = "github.com/cloudquery/cq-provider-aws/client.IgnoreAccessDeniedServiceDisabled"
+    path = "github.com/cloudquery/cq-provider-aws/client.IgnoreCommonErrors"
```

Comment:
use `IgnoreCommonErrors` for the block name as well

Reply:
Done 👍